### PR TITLE
Carbon Black Fix bug

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28219

## Description
**Integration:** VMware Carbon Black EDR (Live Response API)
**Command:** cb-push-file-to-endpoint
**Description:** When using this command to retrieve a big file, The command doesn't return a valid response, the user expects to get the file from the endpoint.

##### VMware Carbon Black EDR (Live Response API)
- Remove deprecate from 'cb-file-get' command.
- Add argument 'download' to 'cb-get-file-from-enpoint', True for downloading the file and False for just uploading it to Carbon black (Will not BC, By default True). Will be used for large files download. 

##### New: Carbon Black Live Response - Create active session
- Create an active session, If the active session already will use the existing session.
##### New: Carbon Black Live Response - Wait until command complete
- Polling command status until finished with an error or complete.
##### New: Carbon Black Live Response - Download file
- Download file from a sensor.

## Screenshots
image

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Carbon_Black_Enterprise_Live_Response/Playbooks/playbook-Carbon_Black_Live_Response_-_Download_file.yml [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    type: regular
+    task:
+      id: f1f386ea-a764-40b2-853d-f39aeef99fd8
+      version: -1
+      name: Close session
+      description: Close the given session
+      script: carbonblackliveresponse|||cb-session-close
+      type: regular
+      iscommand: true
+      brand: carbonblackliveresponse
```
Comment (yaakovi) [inline Packs/Carbon_Black_Enterprise_Live_Response/Playbooks/playbook-Carbon_Black_Live_Response_-_Download_file.yml:220]:
```suggestion
      script: "|||cb-session-close"
      type: regular
      iscommand: true
      brand: ""
```

File: Packs/Carbon_Black_Enterprise_Live_Response/Playbooks/playbook-Carbon_Black_Live_Response_-_Create_active_session.yml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: sensor
+  value:
+    simple: |2+
+
```
Comment (yaakovi) [inline Packs/Carbon_Black_Enterprise_Live_Response/Playbooks/playbook-Carbon_Black_Live_Response_-_Create_active_session.yml:243]: ?

**Review by yaakovi — CHANGES_REQUESTED** [review]
let's do a short demo (want to see the playbook runs)

**Comment by GalRabin** [comment]
@yaakovi @EdenOhana 
Fixed all review comment, Validated with 63 MB file.

**Comment by Itay4** [comment]
@kirbles19 are you done here? is this docs approved?

File: Packs/Carbon_Black_Enterprise_Live_Response/Playbooks/playbook-Carbon_Black_Live_Response_-_Download_file.yml [hunk 3]
```diff
@@ -33,23 +33,24 @@ tasks:
...
     id: "1"
-    taskid: 3b8cf07f-3e41-46b6-8c6c-08cf1a29f358
+    taskid: 2634dc8a-846d-421d-886f-3ea11b43a821
     type: playbook
     task:
-      id: 3b8cf07f-3e41-46b6-8c6c-08cf1a29f358
+      id: 2634dc8a-846d-421d-886f-3ea11b43a821
       version: -1
       name: Carbon Black Live Response - Create active session
+      description: Create an active sessio.
```
Comment (yaakovi) [inline Packs/Carbon_Black_Enterprise_Live_Response/Playbooks/playbook-Carbon_Black_Live_Response_-_Download_file.yml:42]:
```suggestion
      description: Create an active session.
```

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

**Pull request merged**